style.css

```diff
@@ -0,0 +1,201 @@
+  .wrap {
+    max-width: 1920px;
+    min-width: 375px;
+    height: 100vh;
+    margin: 0 auto;
+    background: url(./img/backround.jpg) no-repeat;
+    background-size: cover;
+    font-family: Museo;
+  }
+
+  .header {
+    width: 85%;
+    height: 200px;
+    margin: 0 auto;
+    display: flex;
+    align-items: center;
+  }
+
+  .header__logo {
+    justify-content: center;
+  }
+
+  .space {
+    flex: 2;
+  }
+
+  .menu {
+    flex: 2;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .menu__link {
+    color: #f8eee4;
+    text-decoration: none;
+    font-weight: bold;
+  }
+
+  .menu__link:hover {
+    color: grey;
+  }
+
+  @media all and (max-width: 600px) {
+    .menu__link {
+      visibility: hidden;
+    }
+  }
+
+  .burger {
+    width: 25px;
+    height: 14px;
+    position: relative;
+  }
+
+  .burger .bar {
+    padding: 0;
+    width: 25px;
+    height: 3px;
+    background-color: white;
+    display: block;
+    position: absolute;
+  }
+
+  .bar1 {
+    top: 0;
+  }
+
+  .bar2 {
+    top: 5px;
+  }
+
+  .bar3 {
+    bottom: 0;
+  }
+
+  @media all and (min-width: 600px) {
+    .burger {
+      display: none;
+    }
+  }
+
+  .main-wrap {
+    display: flex;
+  }
+
+  .article {
+    height: 60vh;
+    display: flex;
+    align-items: center;
+    flex: 2;
+    justify-content: center;
+    min-width: 380px;
+    min-height: 230px;
+
+  }
+
+  .article__content {
+    max-width: 641px;
+  }
+
+  .content__Bali {
+    height: 185px;
+    display: flex;
+    justify-content: flex-end;
+    background: url(img/Bali.png) no-repeat;
+    align-items: flex-end;
+    background-size: 99%;
+  }
+
+  .indonesia {
+    width: 90%;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/VeronikasBaliProject/pullRequest?prid=5927042)